Introduction/tutorials/Introduction.md

```diff
@@ -325,7 +325,7 @@ Galaxy makes this very easy with the `Extract workflow` option. This means any t
 >
 >    ![](../images/history_menu_extract_workflow.png)
 >
->    The center pane will change and you will be able to choose which steps to include/exclude and how to name the newly created workflow.
+>    The center panel will change and you will be able to choose which steps to include/exclude into your workflow, and how to name the newly created one.
```

Comment:
What about "...and you will be able to choose a name for the new workflow and which steps to include." ?